app/assets/stylesheets/puppetclasses.scss

```diff
@@ -2,5 +2,26 @@
   .fields {
     border-top: 1px dotted lightgrey;
     padding-top: 10px;
+    &:nth-child(3) {
+      border-top: none;
+    }
   }
 }
+
+.lookup-keys-container {
+  h6 {
+    margin-top: 0px;
+  }
+
+  .help-block {
+    margin: 0px;
+  }
+
+  input[type=checkbox]{
+    margin-top: 11px;
+  }
+}
+
+.btn-fullscreen{
```

Comment:
Agreed, I'll move this as I need it for both places you mentioned and it doesn't affect provision templates since there is no button close to the full screen one on the right side.